Source/WebCore/editing/TextIterator.cpp

```diff
@@ -1019,8 +1003,11 @@ bool TextIterator::shouldRepresentNodeOffsetZero()
     // If this node is unrendered or invisible the VisiblePosition checks below won't have much meaning.
     // Additionally, if the range we are iterating over contains huge sections of unrendered content, 
     // we would create VisiblePositions on every call to this function without this check.
-    if (!currentNode->renderer() || currentNode->renderer()->style().visibility() != Visibility::Visible
-        || (is<RenderBlockFlow>(*currentNode->renderer()) && !downcast<RenderBlockFlow>(*currentNode->renderer()).height() && !is<HTMLBodyElement>(currentNode)))
+    if (!currentNode->renderer() || currentNode->renderer()->style().visibility() != Visibility::Visible)
+        return false;
+
+    if (CheckedPtr renderBlockFlow = dynamicDowncast<RenderBlockFlow>(*currentNode->renderer());
+        renderBlockFlow && !renderBlockFlow->height() && !is<HTMLBodyElement>(currentNode))
```

Comment:
The style checker does not like this, but this seems reasonable to me?

Reply:
The style checker is right, since your condition is on more than one line, we prefer to use curly braces in this case. Please add curly braces.

Reply:
If I make it
```
    if (CheckedPtr renderBlockFlow = dynamicDowncast<RenderBlockFlow>(*currentNode->renderer());
        renderBlockFlow && !renderBlockFlow->height() && !is<HTMLBodyElement>(currentNode)) {
        return false;
    }
```
the style checker is still not happy:
```
ERROR: Source/WebCore/editing/TextIterator.cpp:1009:  Multi line control clauses should use braces.  [whitespace/braces] [4]
```
I didn't see anything in https://webkit.org/code-style-guidelines/#braces that would help me out.